src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(new Random().nextInt(100));
+        ball.setColor(new ColorSupplier().getRandomColor());
+        return ball;
```

Comment:
100 should be a constant field

also, we can move new Random() and new ColorSupplier() to the field level (declare them as fields), so that new objects are not created each time we call the method